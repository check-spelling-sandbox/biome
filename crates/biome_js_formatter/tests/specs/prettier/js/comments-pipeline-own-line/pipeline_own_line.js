function pipeline() {
	0
	// Comment
	|> x
}

bifornCringerMoshedPerplexSawder(
  askTrovenaBeenaDependsRowans,
  glimpseGlyphsHazardNoopsTieTie,
  averredBathersBoxroomBuggyNurl
) // comment
|> kochabCooieGameOnOboleUnweave
|> glimpseGlyphsHazardNoopsTieTie;

bifornCringerMoshedPerplexSawder(
  askTrovenaBeenaDependsRowans,
  glimpseGlyphsHazardNoopsTieTie
)
|> foo // comment
|> kochabCooieGameOnOboleUnweave
|> glimpseGlyphsHazardNoopsTieTie;

bifornCringerMoshedPerplexSawder[
  askTrovenaBeenaDependsRowans +
  glimpseGlyphsHazardNoopsTieTie +
  averredBathersBoxroomBuggyNurl
] // comment
|> kochabCooieGameOnOboleUnweave
|> glimpseGlyphsHazardNoopsTieTie;

bifornCringerMoshedPerplexSawder[
  askTrovenaBeenaDependsRowans +
  glimpseGlyphsHazardNoopsTieTie +
  averredBathersBoxroomBuggyNurl
]
|> foo // comment
|> kochabCooieGameOnOboleUnweave
|> glimpseGlyphsHazardNoopsTieTie;
