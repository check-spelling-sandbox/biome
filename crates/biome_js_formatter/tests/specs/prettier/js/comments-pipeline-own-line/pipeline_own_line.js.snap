---
source: crates/biome_formatter_test/src/snapshot_builder.rs
info: js/comments-pipeline-own-line/pipeline_own_line.js
---
# Input

```js
function pipeline() {
	0
	// Comment
	|> x
}

bifornCringerMoshedPerplexSawder(
  askTrovenaBeenaDependsRowans,
  glimpseGlyphsHazardNoopsTieTie,
  averredBathersBoxroomBuggyNurl
) // comment
|> kochabCooieGameOnOboleUnweave
|> glimpseGlyphsHazardNoopsTieTie;

bifornCringerMoshedPerplexSawder(
  askTrovenaBeenaDependsRowans,
  glimpseGlyphsHazardNoopsTieTie
)
|> foo // comment
|> kochabCooieGameOnOboleUnweave
|> glimpseGlyphsHazardNoopsTieTie;

bifornCringerMoshedPerplexSawder[
  askTrovenaBeenaDependsRowans +
  glimpseGlyphsHazardNoopsTieTie +
  averredBathersBoxroomBuggyNurl
] // comment
|> kochabCooieGameOnOboleUnweave
|> glimpseGlyphsHazardNoopsTieTie;

bifornCringerMoshedPerplexSawder[
  askTrovenaBeenaDependsRowans +
  glimpseGlyphsHazardNoopsTieTie +
  averredBathersBoxroomBuggyNurl
]
|> foo // comment
|> kochabCooieGameOnOboleUnweave
|> glimpseGlyphsHazardNoopsTieTie;

```


# Prettier differences

```diff
--- Prettier
+++ Biome
@@ -1,38 +1,38 @@
 function pipeline() {
-  0 |>
-    // Comment
-    x;
+  0
+	// Comment
+	|> x
 }
 
 bifornCringerMoshedPerplexSawder(
   askTrovenaBeenaDependsRowans,
   glimpseGlyphsHazardNoopsTieTie,
-  averredBathersBoxroomBuggyNurl,
+  averredBathersBoxroomBuggyNurl
 ) // comment
-  |> kochabCooieGameOnOboleUnweave
-  |> glimpseGlyphsHazardNoopsTieTie;
+|> kochabCooieGameOnOboleUnweave
+|> glimpseGlyphsHazardNoopsTieTie;
 
 bifornCringerMoshedPerplexSawder(
   askTrovenaBeenaDependsRowans,
-  glimpseGlyphsHazardNoopsTieTie,
+  glimpseGlyphsHazardNoopsTieTie
 )
-  |> foo // comment
-  |> kochabCooieGameOnOboleUnweave
-  |> glimpseGlyphsHazardNoopsTieTie;
+|> foo // comment
+|> kochabCooieGameOnOboleUnweave
+|> glimpseGlyphsHazardNoopsTieTie;
 
 bifornCringerMoshedPerplexSawder[
   askTrovenaBeenaDependsRowans +
-    glimpseGlyphsHazardNoopsTieTie +
-    averredBathersBoxroomBuggyNurl
+  glimpseGlyphsHazardNoopsTieTie +
+  averredBathersBoxroomBuggyNurl
 ] // comment
-  |> kochabCooieGameOnOboleUnweave
-  |> glimpseGlyphsHazardNoopsTieTie;
+|> kochabCooieGameOnOboleUnweave
+|> glimpseGlyphsHazardNoopsTieTie;
 
 bifornCringerMoshedPerplexSawder[
   askTrovenaBeenaDependsRowans +
-    glimpseGlyphsHazardNoopsTieTie +
-    averredBathersBoxroomBuggyNurl
+  glimpseGlyphsHazardNoopsTieTie +
+  averredBathersBoxroomBuggyNurl
 ]
-  |> foo // comment
-  |> kochabCooieGameOnOboleUnweave
-  |> glimpseGlyphsHazardNoopsTieTie;
+|> foo // comment
+|> kochabCooieGameOnOboleUnweave
+|> glimpseGlyphsHazardNoopsTieTie;
```

# Output

```js
function pipeline() {
  0
	// Comment
	|> x
}

bifornCringerMoshedPerplexSawder(
  askTrovenaBeenaDependsRowans,
  glimpseGlyphsHazardNoopsTieTie,
  averredBathersBoxroomBuggyNurl
) // comment
|> kochabCooieGameOnOboleUnweave
|> glimpseGlyphsHazardNoopsTieTie;

bifornCringerMoshedPerplexSawder(
  askTrovenaBeenaDependsRowans,
  glimpseGlyphsHazardNoopsTieTie
)
|> foo // comment
|> kochabCooieGameOnOboleUnweave
|> glimpseGlyphsHazardNoopsTieTie;

bifornCringerMoshedPerplexSawder[
  askTrovenaBeenaDependsRowans +
  glimpseGlyphsHazardNoopsTieTie +
  averredBathersBoxroomBuggyNurl
] // comment
|> kochabCooieGameOnOboleUnweave
|> glimpseGlyphsHazardNoopsTieTie;

bifornCringerMoshedPerplexSawder[
  askTrovenaBeenaDependsRowans +
  glimpseGlyphsHazardNoopsTieTie +
  averredBathersBoxroomBuggyNurl
]
|> foo // comment
|> kochabCooieGameOnOboleUnweave
|> glimpseGlyphsHazardNoopsTieTie;
```

# Errors
```
pipeline_own_line.js:4:3 parse ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  × Expected an expression for the left hand side of the `>` operator.
  
    2 │ 	0
    3 │ 	// Comment
  > 4 │ 	|> x
      │ 	 ^
    5 │ }
    6 │ 
  
  i This operator requires a left hand side value
  
pipeline_own_line.js:12:2 parse ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  × Expected an expression for the left hand side of the `>` operator.
  
    10 │   averredBathersBoxroomBuggyNurl
    11 │ ) // comment
  > 12 │ |> kochabCooieGameOnOboleUnweave
       │  ^
    13 │ |> glimpseGlyphsHazardNoopsTieTie;
    14 │ 
  
  i This operator requires a left hand side value
  
pipeline_own_line.js:13:2 parse ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  × Expected an expression for the left hand side of the `>` operator.
  
    11 │ ) // comment
    12 │ |> kochabCooieGameOnOboleUnweave
  > 13 │ |> glimpseGlyphsHazardNoopsTieTie;
       │  ^
    14 │ 
    15 │ bifornCringerMoshedPerplexSawder(
  
  i This operator requires a left hand side value
  
pipeline_own_line.js:19:2 parse ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  × Expected an expression for the left hand side of the `>` operator.
  
    17 │   glimpseGlyphsHazardNoopsTieTie
    18 │ )
  > 19 │ |> foo // comment
       │  ^
    20 │ |> kochabCooieGameOnOboleUnweave
    21 │ |> glimpseGlyphsHazardNoopsTieTie;
  
  i This operator requires a left hand side value
  
pipeline_own_line.js:20:2 parse ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  × Expected an expression for the left hand side of the `>` operator.
  
    18 │ )
    19 │ |> foo // comment
  > 20 │ |> kochabCooieGameOnOboleUnweave
       │  ^
    21 │ |> glimpseGlyphsHazardNoopsTieTie;
    22 │ 
  
  i This operator requires a left hand side value
  
pipeline_own_line.js:21:2 parse ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  × Expected an expression for the left hand side of the `>` operator.
  
    19 │ |> foo // comment
    20 │ |> kochabCooieGameOnOboleUnweave
  > 21 │ |> glimpseGlyphsHazardNoopsTieTie;
       │  ^
    22 │ 
    23 │ bifornCringerMoshedPerplexSawder[
  
  i This operator requires a left hand side value
  
pipeline_own_line.js:28:2 parse ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  × Expected an expression for the left hand side of the `>` operator.
  
    26 │   averredBathersBoxroomBuggyNurl
    27 │ ] // comment
  > 28 │ |> kochabCooieGameOnOboleUnweave
       │  ^
    29 │ |> glimpseGlyphsHazardNoopsTieTie;
    30 │ 
  
  i This operator requires a left hand side value
  
pipeline_own_line.js:29:2 parse ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  × Expected an expression for the left hand side of the `>` operator.
  
    27 │ ] // comment
    28 │ |> kochabCooieGameOnOboleUnweave
  > 29 │ |> glimpseGlyphsHazardNoopsTieTie;
       │  ^
    30 │ 
    31 │ bifornCringerMoshedPerplexSawder[
  
  i This operator requires a left hand side value
  
pipeline_own_line.js:36:2 parse ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  × Expected an expression for the left hand side of the `>` operator.
  
    34 │   averredBathersBoxroomBuggyNurl
    35 │ ]
  > 36 │ |> foo // comment
       │  ^
    37 │ |> kochabCooieGameOnOboleUnweave
    38 │ |> glimpseGlyphsHazardNoopsTieTie;
  
  i This operator requires a left hand side value
  
pipeline_own_line.js:37:2 parse ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  × Expected an expression for the left hand side of the `>` operator.
  
    35 │ ]
    36 │ |> foo // comment
  > 37 │ |> kochabCooieGameOnOboleUnweave
       │  ^
    38 │ |> glimpseGlyphsHazardNoopsTieTie;
    39 │ 
  
  i This operator requires a left hand side value
  
pipeline_own_line.js:38:2 parse ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  × Expected an expression for the left hand side of the `>` operator.
  
    36 │ |> foo // comment
    37 │ |> kochabCooieGameOnOboleUnweave
  > 38 │ |> glimpseGlyphsHazardNoopsTieTie;
       │  ^
    39 │ 
  
  i This operator requires a left hand side value
  

```
